### ISSUE TITLE
No doc, no composer

### ISSUE THREAD

**Issue by tango-charly** [anouarcharif/CakePHP3-Media#1]
Salut @anouarcharif 

Bonne initiative, mais pas d'installation avec composer, doc manquante, et référence à Grafikart oublié.
:dizzy_face:

**Comment by anouarcharif** [comment]
Bonjour @tango-charly

Mais si, j'ai une documentation pour ça : http://anouarcharif.github.io/CakePHP3-Media/index.html

Oui pour l'idée je peux mettre Grafikart comme référence, mais pas pour le code 

Merci de votre passage ...
:o2:

**Comment by tango-charly** [comment]
La doc est bien faite, il faut l'inclure le lien dans le ReadMe.
Après lecture du code, on voit bien que c'est une migration du Plugin de Grafikart pour cake 3.0, c'est une bonne idée, je trouve, en revanche il manque encore des choses.
La première, l'installation avec composer ! Pour cake 3.0 c'est devenu incontournable

**Comment by anouarcharif** [comment]
Bonjour,

Merci de votre passage ... je suis ouvert à toute proposition pour amélioration de ce plugin ...

Je vais faire les tests pour composer ...

**Comment by tango-charly** [comment]
Ok, ça marche, bon je l'ai toujours pas testé.
Je ferai peut être une pull request quand j’aurai le temps.

Premier conseil, mets Media au pluriel, pour respecter les nouvelles conventions de Cake

**Comment by anouarcharif** [comment]
Bonjour,

Ok, je vous remercie de votre passage ...

Je reste à l'écoute ...

Amicalement ...

**Comment by tango-charly** [comment]
Et enregistres le nom des Entités au pluriel en base, pour éviter de faire : $ref = Inflector::pluralize($ref);

**Comment by anouarcharif** [comment]
Très bonne remarque ... ceci dit que l'utilisateur doit passer le nom d'entité en pluriel à l'appel de l'iframe ... 

Je vais modifier le code et la documentation pour intégrer ça ...

**Comment by tango-charly** [comment]
A l'appelle de l'iframe, c'est l'utilisateur qui mets le nom de l'identité liée, non ?

**Comment by anouarcharif** [comment]
Oui, le nom de l'entité doit être en pluriel (les standars de CakePHP 3) ... l'entité liée est chargé automatiquement dans le controller ... 

J'ai mis pluralize pour que l'utilisateur utilise le singulier dans l'appel à l'iframe c'est tout ...

**Comment by elarion83** [comment]
Salut Salut ! Dis-moi, j'ai une erreur pour le chargement du helper, cakephp ne me trouve pas le MediaHelper et me génère donc une erreur.. as-tu une idée d'ou cela peut provenir ?

Merci ! ^^
